Prace_domowe/Praca_domowa2/Jamtozy_Bartosz/pd2_allegro.ipynb

```diff
@@ -0,0 +1,2536 @@
+{
```

Comment:
<p>Oprócz podawania odch. stand. warto byłoby jeszcze wyliczyć średnią (odnosi się to do wszystkich analiz)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/67/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>